.github/actions/make_release/action.yml

```diff
@@ -1,6 +1,8 @@
+---
 name: 'Make Release'
 description: 'Make a release when requested'
 author: 'Brian J. Murrell'
 runs:
   using: 'docker'
   image: 'Dockerfile'
+...
```

Comment:
Yaml documents do, to mark the start and end yes.  It may be that use of documents is optional and we could disable this check in the config file, I was just looking for the path of least resistance but could take them out again.

Reply:
I'm not set either way, but it does add some extra clutter to the file, without a clear benefit.

Reply:
Works for me. Thanks!